### PR TITLE
HEEDLS-686 PRN on my account - Fix styling

### PR DESCRIPTION
**Pull request by wilkinsm**
### JIRA link
[https://softwiretech.atlassian.net/browse/HEEDLS-686](https://softwiretech.atlassian.net/browse/HEEDLS-686)

### Description
Fixed styling issues on the PRN fields.
- Labels were styled as hints - fixed
- PRN hint hidden from view unless "yes" was selected
- Double left border when an error occured with the PRN value

### Screenshots
image

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors.
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [x] Manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme. Links (if any) are below:
  - [doc_1_here](link_1_here)
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [x] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

**Review by livzorn — CHANGES_REQUESTED** [review]
Looks good, but I gave it a quick test and noticed the error messages regarding the PRN all end in full stops, which isn't consistent with our other error messages. can't comment against those lines but they're in the ProfessionalRegistrationNumberHelper

**Review by livzorn — APPROVED** [review]
Looks good :)

File: DigitalLearningSolutions.Web.Tests/Helpers/ProfessionalRegistrationNumberHelperTests.cs [hunk 1]
```diff
@@ -105,25 +105,25 @@ public void ValidateProfessionalRegistrationNumber_sets_error_when_hasPrn_is_not
...
-        [TestCase("123", "Professional registration number must be between 5 and 20 characters.")]
-        [TestCase("0123456789-0123456789", "Professional registration number must be between 5 and 20 characters.")]
+        [TestCase(null, "Enter professional registration number")]
+        [TestCase("", "Enter professional registration number")]
+        [TestCase("123", "Professional registration number must be between 5 and 20 characters")]
+        [TestCase("0123456789-0123456789", "Professional registration number must be between 5 and 20 characters")]
         [TestCase(
             "01234_",
-            "Invalid professional registration number format. Only alphanumeric (a-z, A-Z and 0-9) and hyphens (-) allowed."
+            "Invalid professional registration number format. Only alphanumeric (a-z, A-Z and 0-9) and hyphens (-) allowed"
```
Comment (SteveJacksonSoft) [inline DigitalLearningSolutions.Web.Tests/Helpers/ProfessionalRegistrationNumberHelperTests.cs:118]: Hmm, seems strange to have one sentence with a full stop and one without. Could we replace it with a "-" or a ":"?

**Review by SteveJacksonSoft — COMMENTED** [review]
Just one thought about error messages - no rereview required.

File: DigitalLearningSolutions.Web.Tests/Helpers/ProfessionalRegistrationNumberHelperTests.cs (hunk 1 above)
Comment (wilkinsm) [reply DigitalLearningSolutions.Web.Tests/Helpers/ProfessionalRegistrationNumberHelperTests.cs]: done

**Pull request merged**